api/src/main/java/org/infinispan/api/common/KeyValueEntry.java

```diff
@@ -0,0 +1,11 @@
+package org.infinispan.api.common;
+
+/**
+ * @param <K>
+ * @param <V>
+ */
+public interface KeyValueEntry<K, V> {
```

Comment:
I know we don't want to reference the `Map` class but would `Map.Entry` not be an okay usage instead?

Reply:
:+1: